modules/README.markdown

```diff
@@ -91,4 +91,39 @@ File Name | Description
 
 File Name | Description
 --------- | -----------
-`.lfrbuild-missing-resources-importer` | Added only on the `master `branch, prevents the theme from theme from being published in case it doesn't contain the *Resources Importer* files.
\ No newline at end of file
+`.lfrbuild-missing-resources-importer` | Added only on the `master `branch, prevents the theme from theme from being published in case it doesn't contain the *Resources Importer* files.
+
+## Source Formatting
+
+### Gradle files
+
+* Always use double quotes, unless necessary.
```

Comment:
The idea is to always use double quotes, unless it's necessary to use single quotes.